lessons/tests.py

```diff
@@ -1,3 +0,0 @@
-from django.test import TestCase
-
-# Create your tests here.
```

Comment:
Did you delete this on purpose?

Reply:
Yep -- the guidance as i understand it is to split tests out into multiple files in a `tests/` directory once you have enough of them. we only have one test file here right now, but the goal is to have more. some details here: https://docs.djangoproject.com/en/2.2/topics/testing/overview/#writing-tests